src/javascript/SelectorTypes/MultipleLeftRightSelector/MultipleLeftRightSelector.jsx

```diff
@@ -29,9 +29,11 @@ export const MultipleLeftRightSelector = ({field, onChange, value}) => {
 
     const readOnly = field.readOnly || field.valueConstraints.length === 0;
 
+    const selectedLabel = arrayValue.length !== 1 ? t(`${labelBase}.selected_plural`, {count: arrayValue.length}) : t(`${labelBase}.selected`, {count: arrayValue.length});
```

Comment:
That test is probably not needed, _plural is already added by i18next based on count :
```suggestion
    const selectedLabel = t(`${labelBase}.selected`, {count: arrayValue.length});
```